### PR TITLE
Fixed texts in FAQ

### PR DESCRIPTION
**Pull request by biodranik**
@laralem Can you please take a look at Portuguese strings? We don't have here pt_BR yet.

### CODE REVIEW

File: data/faq.html [hunk 1]
```diff
@@ -745,20 +700,15 @@ <h3 lang="pl">Nie mogę pobrać (zaktualizować) map</h3>
...
 	<p>Причиной может быть временная проблема с сетью, либо специфические настройки
-	интернет-провайдера<sup><a href="#rlink">[*]</a></sup> или роутера. Пожалуйста, убедитесь, что на 
+	интернет-провайдера или роутера. Пожалуйста, убедитесь, что на
 	устройстве достаточно свободного места для загрузки и хранения карт, попробуйте еще раз 
 	через некоторое время, или подключитесь к другой точке Wi-Fi.</p>
 
-	<p>Если у вас Android 6 или новее, пожалуйста, убедитесь, что приложению разрешен доступ к памяти 
-	устройства, откройте настройки Android → Приложения → Все → Organic Maps → Разрешения → Память.</p>
+	<p>Если у вас Android, пожалуйста, убедитесь, что приложению разрешен доступ к Интернет. Если у вас установлен
+	файрвол/блокировщик подключений, так же разрешите доступ к Интернету системной программе Download Provider.</p>
```
Comment (rtsisyk) [inline data/faq.html:708]:
```suggestion
	файрвол/блокировщик подключений, так же разрешите доступ к Интернету системной программе Android Download Manager.</p>
```

File: data/faq.html [hunk 2]
```diff
@@ -781,9 +730,7 @@ <h3 lang="pl">Nie mogę pobrać (zaktualizować) map</h3>
 	específicos del proveedor de la red o del rúter. Inténtalo de nuevo más tarde o utiliza otro
 	punto de acceso mediante wifi.</p>
 
-	<p>Si tienes Android 6 o posterior, verifica por favor que se ha concedido un permiso para acceder 
-	la memoria para Organic Maps (usualmente, está habilitado de forma predeterminada) en los ajustes de 
-	tu dispositivo → Aplicaciones → Organic Maps → Permisos → Almacenamiento.</p>
+	<p>En Android, asegúrese de haber otorgado acceso a la red para Organic Maps y Download Provider.</p>
```
Comment (rtsisyk) [inline data/faq.html:733]:
```suggestion
	<p>En Android, asegúrese de haber otorgado acceso a la red para Organic Maps y Android Download Manager.</p>
```

File: data/faq.html [hunk 3]
```diff
@@ -767,9 +717,8 @@ <h3 lang="pl">Nie mogę pobrać (zaktualizować) map</h3>
 	Netzwerkanbieter-/Routereinstellungen fehlschlagen. Bitte versuchen Sie es später noch einmal oder
 	verwenden Sie einen anderen WLAN-Access Point.</p>
 
-	<p>Wenn Sie Android 6 oder höher haben, stellen Sie bitte sicher, dass Sie die Speicherberechtigung 
-	für Organic Maps (normalerweise standardmäßig aktiviert) in den Geräteeinstellungen → Apps → Organic Maps → 
-	Berechtigungen → Speicher erteilt haben.</p>
+	<p>Stellen Sie unter Android sicher, dass Sie den Netzwerkzugriff für Organic Maps und den Download-Provider gewährt
```
Comment (rtsisyk) [inline data/faq.html:720]:
```suggestion
	<p>Stellen Sie unter Android sicher, dass Sie den Netzwerkzugriff für Organic Maps und den Android Download Manager gewährt
```

File: data/faq.html [hunk 4]
```diff
@@ -797,9 +744,7 @@ <h3 lang="pl">Nie mogę pobrać (zaktualizować) map</h3>
 	différent. En complément, assurez vous d'avoir suffisamment d'espace disponible pour télécharger 
 	les cartes.</p>
 
-	<p>Si vous avez Android 6 ou supérieur, assurez-vous que vous avez accordé l'autorisation de 
-	stockage pour Organic Maps (généralement activé par défaut) dans les paramètres de votre appareil → 
-	Applications → Organic Maps → Permissions → Stockage.</p>
+	<p>Sur Android, assurez-vous d'avoir autorisé l'accès au réseau pour Organic Maps et Download Provider.</p>
```
Comment (rtsisyk) [inline data/faq.html:747]:
```suggestion
	<p>Sur Android, assurez-vous d'avoir autorisé l'accès au réseau pour Organic Maps et Android Download Manager.</p>
```

File: data/faq.html [hunk 5]
```diff
@@ -826,9 +769,7 @@ <h3 lang="pl">Nie mogę pobrać (zaktualizować) map</h3>
 	ustawienia dostawcy / routera. Spróbuj ponownie później lub użyj innego punktu dostępu Wi-Fi.
 	Ponadto upewnij się, że masz wystarczającą ilość wolnego miejsca do pobrania map.</p>
 
-	<p>Jeśli masz Android 6 lub nowszy, upewnij się, że masz uprawnienia do przechowywania
-	Organic Maps (zazwyczaj jest domyślnie włączony) w ustawieniach urządzenia → Aplikacje → Wszystkie
-	aplikacje → Google Play Store → Zezwolenia</p>
+	<p>W systemie Android upewnij się, że przyznałeś dostęp do sieci dla organicznych map i Download Provider.</p>
```
Comment (rtsisyk) [inline data/faq.html:772]:
```suggestion
	<p>W systemie Android upewnij się, że przyznałeś dostęp do sieci dla organicznych map i Android Download Manager.</p>
```

**Review by rtsisyk — APPROVED** [review]

**Comment by laralem** [comment]
> @laralem Can you please take a look at Portuguese strings? We don't have here pt_BR yet.

@biodranik You mean this file: https://github.com/organicmaps/organicmaps/blob/master/data/faq.html ? And what HTML string should I use for Brazilian Portuguese? For example:
`<h2 lang="pt">FAQ</h2>` and 1 of the following:
`<h2 lang="pt-br">FAQ</h2>`
`<h2 lang="pt-BR">FAQ</h2>`
`<h2 lang="pt_br">FAQ</h2>`
`<h2 lang="pt_BR">FAQ</h2>` <-by you comment this seems the correct one

**Comment by biodranik** [comment]
You should check in your browser's Developer Console by typing there `navigator.language`. I assume that the correct variant is pt-BR or pt-br.

**Pull request merged**